### ISSUE TITLE
Warning when no queues are defined

### ISSUE THREAD

**Issue by charmpitz** [odolbeau/rabbit-mq-admin-toolkit#45]
https://github.com/odolbeau/rabbit-mq-admin-toolkit/blob/beeb950dfa762f75dbbb680f97ac1ebb65ec643d/src/Bab/RabbitMq/Specification/DeadLetterExchangeCanBeCreated.php#L13

```
In DeadLetterExchangeCanBeCreated.php line 13:
                                                                                       
  Warning: count(): Parameter must be an array or an object that implements Countable
```

**Comment by odolbeau** [comment]
Hi @charmpitz 
Could you please give me a bit of context?
What are you trying to achieve? What's your configuration?
Also, if you already have an idea to fix this issue, don't hesitate to create a PR.
Thanks.

**Comment by charmpitz** [comment]
Sure, the idea is that `$config['queues']` is not an array if you don't define anything inside the yaml file under the `queues` node so you cannot call `count` on it hence the warning. I'll try to do a PR for it.

**Comment by stof** [comment]
but if you don't define anything, `!isset($config['queues'])` should catch that.

**Comment by charmpitz** [comment]
Yes but probably when parsing the yaml file instead of setting it as `null` it is set as empty string. So in that case it fails. I haven't checked the value inside but the warning is real so something wrong is happening there.

**Comment by charmpitz** [comment]
This should do it https://github.com/odolbeau/rabbit-mq-admin-toolkit/pull/46/files

**Comment by odolbeau** [comment]
Fixed by #46
